lib/lumberjack/client.rb

```diff
@@ -69,10 +67,9 @@ def initialize(opts={})
         :port => 0,
         :address => "127.0.0.1",
         :ssl_certificate => nil,
-        :window_size => 5000
       }.merge(opts)
       @host = @opts[:address]
-      @window_size = @opts[:window_size]
+      @window_size = 1
```

Comment:
Yeah I understand the need, I wanted to fix the possible data loss by adding ack on every messages.. But I agree the bulk send is a better solution (which isn't supported at the moment on the client, I don't think it would be that hard since the protocol support it.)

Concerning the buffering I think it should belongs in the `logstash-output-lumberjack` instead of the library, like we do in the elasticsearch output.